samples/python/volume.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import cv2 as cv
+import quaternion
```

Comment:
> `import quaternion`

What is that?
We should not require libraries besides of standard Python distribution.

Reply:
write quaternion calculation from core module